@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -40,15 +44,41 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
     initialState: () => ({
       container: React.createRef(),
       rfb: undefined,
+      rfbConnected: false,
     }),
     effects: {
-      initialize: async function () {
+      initialize: function () {
+        this.effects._connect()
+      },
+      _handleDisconnect: function () {
+        this.state.rfbConnected = false
+        let tries = 0
+        const intVal = setInterval(() => {
+          if (this.state.rfbConnected) {
+            clearInterval(intVal)
+            return
+          }
+          if (++tries === 10) {
```

Comment:
You could use [`iterable-backoff`](https://www.npmjs.com/package/iterable-backoff) so that you can wait for VMs that take longer than 10s.